### DISCUSSION TITLE
Any support for R36.2.0 ?

### DISCUSSION THREAD

**Discussion by noob502** [OE4T/meta-tegra#1641] [General]
meta-tegra is released R36.3.0, but is there any plan for R36.2.0 ?

**Comment by ichergui** [comment]
Hi @noob502 
There is no plan regarding L4T R36.2.0 since it was a DP (Developer Preview) release

**Reply by ichergui** [reply]
@noob502 You should use L4T R36.3.0 
Kindly check our [wiki page](https://github.com/OE4T/meta-tegra/wiki/Which-branch-should-I-use%3F) to use the appropriate branch
